contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -2164,7 +2188,16 @@ static List
 				appendStringInfo(&query, "GRANT dummy ON ALL TABLES IN SCHEMA dummy TO dummy WITH GRANT OPTION; ");
 			else
 				appendStringInfo(&query, "GRANT dummy ON ALL TABLES IN SCHEMA dummy TO dummy; ");
-			appendStringInfo(&query, "ALTER DEFAULT PRIVILEGES IN SCHEMA dummy GRANT dummy ON TABLES TO dummy; ");
+			
+			if (owner_other_than_dbo)
+			{
+				/* Grant ALTER DEFAULT PRIVILEGES on schema owner and dbo user. */
+				appendStringInfo(&query, "ALTER DEFAULT PRIVILEGES FOR ROLE dummy IN SCHEMA dummy GRANT dummy ON TABLES TO dummy; ");
+				appendStringInfo(&query, "ALTER DEFAULT PRIVILEGES FOR ROLE dummy IN SCHEMA dummy GRANT dummy ON TABLES TO dummy; ");
```

Comment:
We can improve the performance by executing single alter def. privileges stmt for multiple roles in single go. Reference : https://www.postgresql.org/docs/current/sql-alterdefaultprivileges.html
```
[ FOR { ROLE | USER } target_role [, ...] ]
```